test/utils/MiltonUtils.sol

```diff
@@ -411,4 +421,4 @@ contract MiltonUtils is Test {
             expectedCloserUserUnderlyingTokenBalanceAfterClose;
         return testCaseWhenMiltonLostAndUserEarned;
     }
-}
```

Comment:
You missed one test from old file "should calculate Pay Fixed Position Value - simple case 1" - why?